<launch>
  <node name="main_state" pkg="main_state" type="main.py" output="screen" args="RoboNurse" />
  <node name="gesture_detection" pkg="gesture_detection" type="NIHandtracker" output="screen" >
    <remap from="points" to="/gesture/points"/>
  </node>
  <node ns="door" name="door_detection" pkg="door_detection" type="door_detection_node">
    <remap from="/scan" to="/laser/scan"/>    
  </node>
  <include file="$(find speech_processing)/dic/robo_nurse_2/robo_nurse.launch"/>

</launch>